enterprise/internal/campaigns/syncer.go

```diff
@@ -80,7 +92,14 @@ var (
 // nextSync computes the time we want the next sync to happen.
 func nextSync(h campaigns.ChangesetSyncHeuristics) time.Time {
```

Comment:
Just pointing out again (sorry! 😬) that "Heuristics" is what's happening in `nextSync`. What goes _into_ `nextSync` is the data that's being processed heuristically.

```suggestion
func nextSync(h campaigns.ChangesetSyncData) time.Time {
```